itou/templates/apply/list_of_available_exports.html

```diff
@@ -20,6 +20,7 @@ <h2 class="h1 font-weight-normal text-muted">
 
         <div class="table-responsive">
             <table class="table table-striped table-bordered">
+                <caption>Liste de candidatures reçues</caption>
```

Comment:
```suggestion
                <caption>Liste des candidatures reçues par mois</caption>
```
On affiche des fichiers qui listent les candidatures mois par mois.